test/k8sT/Services.go

```diff
@@ -216,6 +216,16 @@ var _ = Describe("K8sServicesTest", func() {
 			}
 		}
 
+		doRequestsFromOutsideClient := func(url string, count int) {
+			ssh := helpers.GetVagrantSSHMeta(helpers.K8s1VMName())
+			By("Making %d HTTP requests from remote host to %q", count, url)
```

Comment:
nit: `remote host` declares clearly that the source should be another node, but doesn't mention anything about it being outside the cluster. It might be easier to understand failures if this was mentioned either in the `By()` line here or in the `ExpectWithOffset()` below.